tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -107,6 +107,19 @@ public function testSpatialIndex()
         self::assertTrue($indexes['s_index']->hasFlag('spatial'));
     }
 
+    public function testIndexWithLength()
```

Comment:
`: void`

Reply:
Addded in d82b245a25c8dc5d4b216769946e85bf3a1685a4.

I wasn't sure about adding it, because only 7 of the 20 test methods have it. What's the rule here? Keeping the existing methods as they are but adding return types to all new ones?

Reply:
I don't think that we've added `doctrine/coding-standards` to this codebase, but we try to follow it for any new diffs anyway.

In `doctrine/orm`, any diff like this would have led to an automated CS check failure